scripts/change_react_native_version.js

```diff
@@ -2,25 +2,32 @@ const fs = require('fs');
 const path = require('path');
 const https = require('https');
 
-
 async function run() {
   const projectPath = process.argv[2];
   const reactNativeVersion = process.argv[3];
+  const depsSection = process.argv[4] || 'dependencies';
 
   const filePath = path.join(process.cwd(), projectPath, 'package.json');
   console.log(`Trying to change react-native dependency in  ${filePath}`);
 
   let packageJson = require(filePath);
 
-  const data = await fetch(`https://registry.npmjs.org/react-native/${reactNativeVersion}`);
-  const reactVersion = data.peerDependencies.react;
+  if (depsSection === 'devDependencies') {
+    console.log(`Changed dependency:
```

Comment:
Why not `Changed devDependency:`, to be more accurate?

Reply:
Not sure what you mean